src/tools/kata-ctl/src/args.rs

```diff
@@ -145,3 +148,9 @@ pub struct ExecArguments {
     /// kata debug console vport same as configuration, default is 1026.
     pub vport: u32,
 }
+
+#[derive(Debug, Args)]
+pub struct TimeoutArguments {
+    /// timeout value
```

Comment:
I'd drop this comment as the code is clear enough, I think.

Reply:
@jodh-intel I think this appears in the cli output as explanatory comments. 
I would keep the comment and explain what it does here.

Reply:
Oh, if that's the case, then I guess we need to keep it. But please state what unit the timeout value uses (integer seconds?)

Reply:
Maybe I can revert this comment and add timeout unit